tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -1136,14 +1155,32 @@ const reducer = createReducer(
       cardStateMap: nextCardStateMap,
     };
   }),
-  on(actions.stepSelectorToggled, (state, {affordance}) => {
+  on(actions.stepSelectorToggled, (state, {affordance, cardId}) => {
+    const nextCardStateMap = {...state.cardStateMap};
+    if (cardId) {
+      nextCardStateMap[cardId] = {
+        ...nextCardStateMap[cardId],
+        stepSelection: CardSelectionState.DISABLED,
+      };
+    } else {
+      Object.keys(nextCardStateMap).forEach((cardId) => {
+        nextCardStateMap[cardId] = {
+          ...nextCardStateMap[cardId],
+          stepSelection: CardSelectionState.GLOBAL,
```

Comment:
Similar to my comment for rangeSelectionToggled, would be nice to have a comment.